build/patches/Revert-Permit-blocking-of-view-source.patch

```diff
@@ -192,23 +181,11 @@ diff --git a/components/policy/content/policy_blocklist_navigation_throttle.cc b
    if (blocklist_state == URLBlocklistState::URL_IN_ALLOWLIST)
      return PROCEED;
  
-@@ -177,11 +158,6 @@ void PolicyBlocklistNavigationThrottle::OnFirstPoliciesLoadedImpl(
-         ThrottleCheckResult(BLOCK_REQUEST, net::ERR_BLOCKED_BY_ADMINISTRATOR));
-   }
- 
--  if (IsBlockedViewSourceNavigation()) {
--    return CancelDeferredNavigation(
--        ThrottleCheckResult(BLOCK_REQUEST, net::ERR_BLOCKED_BY_ADMINISTRATOR));
--  }
--
-   if (CheckSafeSitesFilter(url).action() == PROCEED)
-     Resume();
- }
 diff --git a/components/policy/content/policy_blocklist_navigation_throttle.h b/components/policy/content/policy_blocklist_navigation_throttle.h
 --- a/components/policy/content/policy_blocklist_navigation_throttle.h
 +++ b/components/policy/content/policy_blocklist_navigation_throttle.h
-@@ -50,10 +50,7 @@ class PolicyBlocklistNavigationThrottle
-   void OnFirstPoliciesLoadedImpl(bool timeout);
+@@ -35,10 +35,7 @@ class PolicyBlocklistNavigationThrottle : public content::NavigationThrottle {
+   const char* GetNameForLogging() override;
  
   private:
 -  // Returns TRUE if this navigation is to view-source: and view-source is on
```

Comment:
a few lines below in the patch
```
+ ThrottleCheckResult GetUrlThrottleResult(const GURL& url);
```
I think it should be eliminated

Reply:
I will remove it but it didn't give any linking error.

Reply:
> give any linking error.

I guess it's because that method is never called by anyone